src/SFA.DAS.EmployerFinance.Web/Filters/UrlsViewBagFilter.cs

```diff
@@ -0,0 +1,34 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Microsoft.Extensions.DependencyInjection;
+using SFA.DAS.EmployerFinance.Web.RouteValues;
+using SFA.DAS.EmployerFinance.Web.Urls;
+using StructureMap;
+
+namespace SFA.DAS.EmployerFinance.Web.Filters
+{
+    public class UrlsViewBagFilter : IAsyncActionFilter
+    {
+        public async Task OnActionExecutionAsync(ActionExecutingContext context, ActionExecutionDelegate next)
+        {
+            var controller = context.Controller as Controller;
+            if (controller == null)
+                return;
```

Comment:
Could curly braces be added here to follow our code conventions?

Reply:
Done.